scripts/start.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/python3
```

Comment:
This should not be needed. It's identical to `vitables/__main__.py`, which can be started simply doing `python -m vitables`.